tests/playwright/sanity/modules/floating-buttons/floating-elements-admin.test.ts

```diff
@@ -0,0 +1,182 @@
+import { parallelTest as test } from '../../../parallelTest';
+import FloatingElementPage from './floating-element-page';
+import { expect } from '@playwright/test';
+import EditorSelectors from '../../../selectors/editor-selectors';
+
+test.describe( 'Verify floating buttons editor, admin page and front page behavior', () => {
+	test.afterEach( async ( { browser, apiRequests }, testInfo ) => {
+		const context = await browser.newContext();
+		const page = await context.newPage();
+		const floatingElPage = new FloatingElementPage( page, testInfo, apiRequests );
+		await floatingElPage.deleteAllFloatingButtons();
+		await page.close();
+	} );
+
+	test( 'Verify editor behavior by creating post through API and also FE behavior', async (
```

Comment:
This test looks quite large and should be broken down into several tests with more test steps inside. 
It's better to divide it into logical parts and use test.step for every part.

Reply:
Please see an example for lines 24-31

Reply:
@Svitlana-Dykun done